cmd/service/utils.go

```diff
@@ -39,26 +53,62 @@ func handleValidationError(err error) {
 	}
 }
 
-func gitClone(repoURL string, path string, message string) error {
-	u, err := url.Parse(repoURL)
-	if err != nil {
-		return err
+// prepareService downloads if needed, create the service, build it and inject configuration
+func prepareService(path string) *service.Service {
```

Comment:
thanks to codeclimate, I think this function is not needed here, same for `downloadServiceIfNeeded` or `injectConfigurationInDependencies` because they moved to the `mesg/deploy` @ilgooz can you confirm ?

Reply:
[yes](https://github.com/mesg-foundation/core/blob/0d43e34/mesg/deploy_deployer.go) that is correct. @krhubert can you please check the current version of cmd/service/utils.go in dev branch and compare with yours to see diffs. Thanks a lot.